lib/instrumentation/modules/pg.js

```diff
@@ -10,12 +10,15 @@ var symbols = require('../../symbols')
 var { getDBDestination } = require('../context')
 
 module.exports = function (pg, agent, { version, enabled }) {
+  if (!enabled) {
```

Comment:
+1 -- this is both more correct and should not effect the functionality of the instrumentation -- when `enabled` was passed on to the `patchClient` method it bailed in a guard clause immediately.